rms_fe/src/index.css

```diff
@@ -0,0 +1,21 @@
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+.login-container {
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.login-wrapper {
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+}
```

Comment:
This can be a css file local to the component, you can just import it inside the home page using the `import cssfile.css` syntax